static/src/javascripts/projects/common/modules/commercial/commercial-feature-policies.js

```diff
@@ -0,0 +1,112 @@
+define([
+    'common/utils/_',
+    'common/utils/config',
+    'common/modules/commercial/user-ad-preference',
+    'common/modules/user-prefs'
+], function (
+    _,
+    config,
+    userAdPreference,
+    userPrefs
+) {
+    var policies = {};
+
+    policies.defaultAds = function () {
+        return new CommercialFeatureSwitches(true);
+    };
+
+    policies.sensitiveContent = function () {
+        // Avoid inappropriate adverts on pages with content of a sensitive nature
+        if (config.page.shouldHideAdverts || config.page.section === 'childrens-books-site') {
+            return new CommercialFeatureSwitches(false);
+        }
+    };
+
+    policies.sslContent = function () {
+        if (config.page.isSSL && config.page.section !== 'admin') {
+            return new CommercialFeatureSwitches(false);
+        }
+    };
+
+    policies.uriSwitches = function () {
+        if (window.location.hash.match(/[#&]noads(&.*)?$/)) {
+            return new CommercialFeatureSwitches(false);
+        }
+    };
+
+    policies.userPrefs = function () {
```

Comment:
I'm not actually sure what the userPrefs module is used for; I'm using it here because the original dfp.js hinged on it. Was it perhaps for testing purposes?